pkg/instance/instance.go

```diff
@@ -87,7 +87,7 @@ type BlockingReason struct {
 
 var (
 	// BlockedLoginFailed is used when a security issue has been detected on the instance
-	BlockedLoginFailed = BlockingReason{Code: "LOGIN_FAILED", Message: "The instance was block because of too many login failed attempts"}
+	BlockedLoginFailed = BlockingReason{Code: "LOGIN_FAILED", Message: "The instance was blocked because of too many login failed attempts"}
```

Comment:
The message is weird: it should be too many login failed or too many login attempts, but not both failed and attempts, no?